apps/web/src/scenes/NftDetailPage/NftDetailPageFallback.tsx

```diff
@@ -3,12 +3,35 @@ import styled from 'styled-components';
 
 import breakpoints from '~/components/core/breakpoints';
 import { HStack, VStack } from '~/components/core/Spacer/Stack';
+import { useNftPreviewFallbackState } from '~/contexts/nftPreviewFallback/NftPreviewFallbackContext';
+
+type Props = {
+  tokenId: string;
+};
+
+export default function NftDetailPageFallback({ tokenId }: Props) {
+  const { cachedUrls } = useNftPreviewFallbackState();
+
+  const hasPreviewUrl = cachedUrls[tokenId]?.type === 'preview';
+  const dimensions = cachedUrls[tokenId]?.dimensions;
 
-export default function NftDetailPageFallback() {
   return (
     <StyledFullPageLoader>
-      <StyledNftSkeleton />
-      <VStack gap={24}>
+      {hasPreviewUrl ? (
+        <VisibilityContainer>
+          <StyledImageWrapper className={'visible'}>
+            <StyledImage
+              src={cachedUrls[tokenId]?.url}
+              height={dimensions?.height ?? 600}
+              width={dimensions?.width ?? 600}
```

Comment:
1. should handle mobile
2. let's import the `600` from wherever we define the const